src/main/java/org/jboss/modules/JDKPaths.java

```diff
@@ -57,22 +57,23 @@
             moduleClass = JDKPaths.class.getClassLoader().loadClass(JDK9_CLASS);
         } catch (final Throwable ignored) {}
         isJDK9orAbove = moduleClass != null;
-        final Set<String> pathSet = new FastCopyHashSet<>(1024);
+        final Set<String> pathSet;
         final Set<String> jarSet = new FastCopyHashSet<>(1024);
         if (isJDK9orAbove) {
+            // TODO: Remove this to the JDK9 supplement
```

Comment:
This whole 'isJDK9orAbove' branch can be eliminated.
I see it is already present in jboss-modules-jdk9-supplement.